react-app-rewired/headers/csps/fiatRamps/onRamper.ts

```diff
@@ -0,0 +1,6 @@
+import type { Csp } from '../../types'
+
+export const csp: Csp = {
+  'frame-src': ['https://widget.onramper.com', 'https://buy.moonpay.com'],
+  'frame-ancestors': ['*']
```

Comment:
we'll want this to be `'self'` https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors#examples